include/kodlab_mjbots_sdk/pi3hat_moteus_interface.h

```diff
@@ -203,7 +211,7 @@ class Pi3HatMoteusInterface : public kodlab::AbstractRealtimeObject{
     input.tx_can = { tx_can_.data(), tx_can_.size() };
     input.rx_can = { rx_can_.data(), rx_can_.size() };
     input.request_attitude = true;
-    input.attitude = &attitude_;
+    input.attitude = attitude_.get();
```

Comment:
@jdcaporale  So right now nothing at the pi3hat level uses smart pointers. I think its fine to keep it that way, or it should be a separate PR